docs_source_files/content/_index.nl.md

```diff
@@ -0,0 +1,148 @@
+from selenium.webdriver.support.expected_conditions import presence_of_element_located
+
+#This example requires Selenium WebDriver 3.13 or newer
+with webdriver.Firefox() as driver:
+    wait = WebDriverWait(driver, 10)
+    driver.get("https://google.com/ncr")
+    driver.find_element_by_name("q").send_keys("cheese" + Keys.RETURN)
+    first_result = wait.until(presence_of_element_located((By.CSS_SELECTOR, "h3>a")))
+    print(first_result.text)
+  {{< / code-panel >}}
+  {{< code-panel language="csharp" >}}
+using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+
+class HelloSelenium
+{
+    static void Main()
+    {
+        using (IWebDriver driver = new FirefoxDriver())
+        {
+            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
+            driver.Navigate().GoToUrl("https://www.google.com/ncr");
+            driver.FindElement(By.Name("q")).SendKeys("cheese" + Keys.Enter);
+            IWebElement firstResult = wait.Until(ExpectedConditions.ElementExists(By.CssSelector("h3>a")));
+            Console.WriteLine(firstResult.Text);
+        }
+    }
+}
+  {{< / code-panel >}}
+  {{< code-panel language="ruby" >}}
+require 'selenium-webdriver'
+
+driver = Selenium::WebDriver.for :firefox
+wait = Selenium::WebDriver::Wait.new(timeout: 10)
+
+begin
+  driver.get 'https://google.com/ncr'
+  driver.find_element(name: 'q').send_keys 'cheese', :return
+  first_result = wait.until { driver.find_element(css: 'h3>a') }
+  puts first_result.text
+ensure
+  driver.quit
+end
+  {{< / code-panel >}}
+  {{< code-panel language="javascript" >}}
+const {Builder, By, Key, until} = require('selenium-webdriver');
+
+(async function example() {
+    let driver = await new Builder().forBrowser('firefox').build();
+    try {
+        await driver.get('https://www.google.com/ncr');
+        await driver.findElement(By.name('q')).sendKeys('cheese', Key.RETURN);
+        let firstResult = await driver.wait(until.elementLocated(By.css('h3>a')),10000);
+        console.log(await firstResult.getText());
+    } finally {
+        await driver.quit();
+    }
+})();
+  {{< / code-panel >}}
+{{< / code-tab >}}
+
+
+See the _[Quick Tour]({{< ref "/getting_started/quick.en.md" >}})_ for a full explanation
+of what goes on behind the scenes when you run this code.
+You should continue on to the [narrative documentation]({{< ref "/introduction/_index.md" >}})
+to understand how you can [install]({{< ref "/selenium_installation/_index.md" >}}) and
+successfully use Selenium as a test automation tool,
+and scaling simple tests like this to run
+in large, distributed environments on multiple browsers,
+on several different operating systems.
+
+## Getting started
+
+If you are new to Selenium,
+we have a few resources that can help you
+get up to speed right away.
+
+* [Quick tour]({{< ref "/getting_started/quick.en.md" >}})
```

Comment:
Shouldn't this one point to the `/getting_started/quick.nl.md` file?

Reply:
Fixed in new push